static/new_home/style.css

```diff
@@ -582,7 +581,7 @@ header
 #donate-us::before{
   display: block;
   content: "";
-  /* margin-top: -25px; */
+  margin-top: -25px;
   height: 25px;
```

Comment:
I think you should change to `65px` here also.

Reply:
> I have my end-sem examinations coming up in a few days and the due dates for all my assignments and projects are also approaching, so I may not be active in reviewing PRs for a few days.

No issues. All the best.